draft-vandijk-dprive-ds-dot-signal-and-pin/draft-vandijk-dprive-ds-dot-signal-and-pin.md

```diff
@@ -96,6 +96,12 @@ If a resolver with support for TBD encounters a DS record with the DNSKEY algori
 It MUST use the hashes attached to the DS records with DNSKEY algorithm type TBD to check whether the public key supplied by the authoritative nameserver is valid.
 If the DoT connection is unsuccessful or the public key supplied the server does not match one of the DS digests, the resolver MUST NOT fall back to unencrypted Do53.
 
+A domain MAY have more than one DS record with DNSKEY algorithm TBD.
+A resolver with support for TBD should then try to verify the public key supplied by the authoritative nameserver against every supplied DS record.
+Multiple records can be used to support multiple DS digest types, multiple TLS key algorithms, different keys for each authoritative, and for key rollovers.
+In case of an algorithm rollover the DS record should be added to all served domains before the new key is deployed on the authoritatives.
```

Comment:
```suggestion
In case of an algorithm or key rollover the new DS record should be added to all served domains before the new key is deployed on the authoritatives.
```